lib/core/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -133,4 +157,8 @@ export class CardViewTextItemComponent implements OnChanges {
             this.cardViewUpdateService.clicked(this.property);
         }
     }
+
+    isChipViewEnabled(): boolean {
```

Comment:
maybe worth turning into the property?
```suggestion
    get isChipViewEnabled(): boolean {
```